docs/operator/deploy-access-kubernetes.md

```diff
@@ -0,0 +1,92 @@
+# Deploying and Accessing Busola in the Kubernetes Cluster
+
+## Architecture
+
+![Busola Kubernetes architecture](assets/busola_kubernetes_3.svg)
+
+To expose Busola, you can use [APIRule](https://github.com/kyma-project/busola/tree/main/resources/istio), [Ingress](https://github.com/kyma-project/busola/tree/main/resources/ingress) or create your own exposing mechanism.
+For more details about environment configuration, see [Environment-Specific Settings](configuration.md#environment-specific-settings).
+
+## Deploying Busola in the Kubernetes Cluster
+
+To install Busola from the release in the Kubernetes cluster, set the `NAMESPACE` and `VERSION` shell environment variables with the desired release and run:
```

Comment:
Why not \*\*NAMESPACE\*\* and \*\*VERSION\*\*?

Reply:
I think I saw them in code font in other docs as well. Shouldn't they be boldfaced?